package.json

```diff
@@ -20,17 +20,17 @@
   "homepage": "https://github.com/cloudbees/blueocean-acceptance-test#readme",
   "devDependencies": {
     "@jenkins-cd/sse-gateway": "0.0.7",
```

Comment:
@tfennelly I was up to update the sse-gateway plugin to 0.0.8, but that breaks the AT. Symptom is that in dev mode you see a ```25189 segmentation fault  nightwatch``` when running nightwatch and in standalone the test case (ff never starts up) simply hang.